pets.json

```diff
@@ -0,0 +1,64 @@
+{
+  "pets": [
+    {
+      "id": 1,
+      "name": "Everly B",
+      "species": "dog",
```

Comment:
We're going to need to do a quick pass to make sure to update `breed` to `species`.